driver/longhorn

```diff
@@ -215,6 +215,7 @@ formatDevIfNeed() {
     local newFsType=$2
     local oldFsType
     local OUT
+    local outputLog="/dev/null"
```

Comment:
Can we output the log to the console instead of `/dev/null`?

E.g. Would that be an impact on kubelet's interpretation of the result return by the flexvolume driver?

Reply:
Yes, we can. flexvolume driver use stdout and stderr to output result. applications use itself stdfiles (stderr,stdout,stdin), they don't affect each other. 
Do we need to instead it ?
How can we receive the log in AWS VM, if use the console ?

Reply:
Sorry, I don't understand. What application you're talking about? As I commented in the issue, I don't think we should just lose the log to /dev/null.

Kubelet is the one invoking the flexvolume driver, I suppose the output of flexvolume driver should be somehow collected by kubelet.

I don't know where the stdout/stderr is for flexvolume driver. That's what I need you to find out. And make sure it won't mess up with the communication between kubelet and flexvolume driver.

Reply:
we can use `docker logs kubelet 2>&1` to show the error's log of flexvolume driver.